pkg/streamingpromql/operators/binary_operation_test.go

```diff
@@ -241,7 +297,292 @@ func TestBinaryOperation_SeriesMerging(t *testing.T) {
 				{Labels: labels.FromStrings("__name__", "right_side", "env", "test", "pod", "i")},
 				{Labels: labels.FromStrings("__name__", "right_side", "env", "test", "pod", "j")},
 			},
-			expectedError: `found duplicate series for the match group {env="test"} on the right side of the operation at timestamp 1970-01-01T00:00:00.002Z: {__name__="right_side", env="test", pod="g"} and {__name__="right_side", env="test", pod="j"}`,
+			expectedError: `found duplicate series for the match group {env="test"} on the right side of the operation at timestamp 1970-01-01T00:00:00.002Z: {__name__="right_side", env="test", pod="g"} and {__name__="right_side", env="test", pod="e"}`,
+		},
+		"mixed float and histogram input series with no conflict": {
```

Comment:
Could you please add some test cases for the scenario where we just have histograms in all series, similar to the test cases for only floats above?